.github/workflows/maven.yml

```diff
@@ -35,13 +35,14 @@ jobs:
         if: contains(matrix.os, 'win') == false
         run: chmod +x ./mvnw
       - name: Build with Maven
-        run: ./mvnw install --file pom.xml --batch-mode
+        # `-DskipTests=true` since they run on `coverage` job.
+        run: ./mvnw install --file pom.xml --batch-mode -DskipTests=true
         env:
           MAVEN_OPTS: -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
       - name: Build Gradle Plugin
         run: cd jte-gradle-plugin && ./gradlew publishToMavenLocal
       - name: Run all the Gradle Plugin tests
-        run: cd test/gradle-test-wrapper && ./gradlew check
+        run: cd test/gradle-test-wrapper && ./gradlew --info check
```

Comment:
`--info` is a good compromise regarding having more information when this step fails and not being super verbose.

Reply:
Nice!